docs/get-started/deployment-and-usage.md

```diff
@@ -426,29 +426,158 @@ configuration for enabling TLS authentication.
 The Common Name (CN) of the nfd-master certificate must match the DNS name of
 the nfd-master Service of the cluster. By default, nfd-master only check that
 the nfd-worker has been signed by the specified root certificate (-ca-file).
-Additional hardening can be enabled by specifying -verify-node-name in
+
+Additional hardening can be enabled by specifying `-verify-node-name` in
 nfd-master args, in which case nfd-master verifies that the NodeName presented
 by nfd-worker matches the Common Name (CN) or a Subject Alternative Name (SAN)
-of its certificate.
+of its certificate.  Note that `-verify-node-name` complicates certificate
+management and is not yet supported in the helm or kustomize deployment methods.
 
 #### Automated TLS certificate management using cert-manager
 
 [cert-manager](https://cert-manager.io/) can be used to automate certificate
 management between nfd-master and the nfd-worker pods.
 
-NFD source code repository contains an example kustomize overlay that can be
-used to deploy NFD with cert-manager supplied certificates enabled. The
-instructions below will install cert-manager and generate a self-signed CA certificate
-and set up cert-manager's
-[CA Issuer](https://cert-manager.io/docs/configuration/ca/) to sign
-`Certificate` requests for NFD components in `node-feature-discovery`
-namespace.
+The NFD source code repository contains an example kustomize overlay and helm
+chart that can be used to deploy NFD with cert-manager supplied certificates enabled.
+
+To install `cert-manager` itself can be done as easily as this, below, or you can
+refer to their documentation for other installation methods such as the helm chart
+they provide.
 
 ```bash
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.1/cert-manager.yaml
+```
+
+To use the kustomize overlay to install node-feature-discovery with TLS enabled, you may use the following:
+
+```bash
 kubectl apply -k deployment/overlays/samples/cert-manager
 ```
 
+To make use of the helm chart, override `values.yaml` to enable both the `tls` and 
```

Comment:
```suggestion
To make use of the helm chart, override `values.yaml` to enable both the `tls` and
```

Fix this and other nits from [mdlint](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_node-feature-discovery/713/pull-node-feature-discovery-verify-docs/1480461362051158016/build-log.txt) to make the CI pass

Reply:
pull-node-feature-discovery-verify-docs now succeeds.